### ISSUE TITLE
Miner projected profits very different than balance in zpool and miningpoolhub

### ISSUE THREAD

**Issue by VictorNegridemos** [aaronsace/MultiPoolMiner#647]
Hello there. I am using mininpoolhub and zpool as pools. After taking notice of projections in the miner and the total value of my balance in miningpoolhub (taking note of exchanges as well) and zpool it came up at only about 40% of what the projections were. This is 3 hours after the 24 hours had passed. I understand time needs to pass before balance is properly updated everywhere so my questions are:

1. Is this normal behavior? Should I be worried about using mpm wrong or not having some file/using wrong settings?
2. What is the normal amount of time I need to wait before my balance in multipoolhub and zpool should should reflect reality with good accuracy?

**Comment by JhonJhonny** [comment]
same here, opened an issue and 1 min ago i posted two screenshots, hope we could solve our problem

**Comment by aaronsace** [comment]
NVidia cards require a lot of switching, try Zpool.

**Comment by VictorNegridemos** [comment]
Hey. I already read that in your previous responses. But it wasn't an answer to either of my 2 questions. Let me rephrase:
1. Are those less than estimate profits a normal thing in the 24h period I described?
2. Considering the oddities of miningpoolhub and it's switching, after how much time can I expect all my balance to be correct? If you don't have an exact answer, try a rough estimate.

And a final one after your response. MPM is telling me miningpoolhub is around 20% more effective than zpool(atm). Do you think because of the switching zpool will still be better?

**Comment by aaronsace** [comment]
40% is poor; you'll never get the projected but it should be more like 70%-80%. 

You need at least 48h to get a 24h reading on MiningPoolHub. 

I find Zpool better for switch because it's PROP instead of PPLNS.

**Comment by VictorNegridemos** [comment]
Thanks for the response. Much obliged.
